omero/developers/Java.txt

```diff
@@ -740,6 +740,31 @@ to an image.
       }
     }
 
+-  **Organize ROIs in Folders.**
+
+::
+    
+    ROIFacility roifac = gateway.getFacility(ROIFacility.class);
+    
+    Collection<ROIData> rois = ...
+    
+    // Add each ROI to a different folder
+    for (ROIData r : rois) {
+        FolderData folder = new FolderData();
+        folder.setName("Folder for ROI " + r.getId());
+        roifac.addRoisToFolders(ctx, imageId, Arrays.asList(r),
```

Comment:
to follow the flow of the page it should be ``image.getId()`` instead of ``imageId``